config/software/make.rb

```diff
@@ -32,8 +32,16 @@
 
   # Work around an error caused by Glibc 2.27
   #
-  # Thanks to: http://www.linuxfromscratch.org/lfs/view/8.2/chapter05/make.html
-  command "sed -i '211,217 d; 219,229 d; 232 d' glob/glob.c", env: env
+  # sed -i is not supported so exclude the patch for FreeBSD, Solaris and AIX
+  unless freebsd? || solaris_11? || aix?
+    if mac_os_x?
+      # OS X ships with BSD sed, where the suffix is mandatory so added sed -i ''
+      command "sed -i '' '211,217 d; 219,229 d; 232 d' glob/glob.c", env: env
```

Comment:
@schisamo mac_os_x patching works with ^ sed -i options, and build and tests pass, so did not exclude the patch on it

Reply:
NM...I misread that `unless` as an `if`. Carryon...nothing to see here. :)

Reply:
@schisamo can you please +1 if it looks good to you